test/Twilio.Test/Twilio.Test.csproj

```diff
@@ -2,28 +2,32 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <RootNamespace>Twilio.Tests</RootNamespace>
-    <TargetFrameworks>netcoreapp2.0;net451;net35</TargetFrameworks>
-    <RuntimeIdentifier Condition=" '$(TargetFramework)' == 'net451' ">win7-x86</RuntimeIdentifier>
+    <TargetFrameworks>net60;net462;net35</TargetFrameworks>
+    <RuntimeIdentifier Condition=" '$(TargetFramework)' == 'net462' ">win7-x86</RuntimeIdentifier>
     <RuntimeIdentifier Condition=" '$(TargetFramework)' == 'net35' ">win7-x86</RuntimeIdentifier>
-    <IsPackable>false</IsPackable>
+    <IsPackable>false</IsPackable>    
+    <GenerateProgramFile>false</GenerateProgramFile>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="coverlet.msbuild" Version="3.1.0">
+    <PackageReference Include="coverlet.msbuild" Version="3.2.0">
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="NUnit" Version="3.6.0" />
-    <PackageReference Include="NSubstitute" Version="2.0.0-rc" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
-    <PackageReference Include="NUnitLite" Version="3.6.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
+    <PackageReference Include="NUnit" Version="3.13.3" />
+    <PackageReference Include="NSubstitute" Version="2.0.3" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.4.2" />
+    <PackageReference Include="NUnitLite" Version="3.13.3" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.6.0" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net451' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net462' ">
     <Reference Include="System.Net" />
     <Reference Include="System.Net.Http" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.19.0" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\src\Twilio\Twilio.csproj" />
+    <ProjectReference Include="..\..\src\Twilio\Twilio.csproj">
+      <SetTargetFramework Condition=" '$(TargetFramework)' == 'net462' ">TargetFramework=netstandard2.0</SetTargetFramework>
```

Comment:
Instructs the test project to use the netstandard2.0 tfm of the Twilio project for tests running on net462. This is only necessary for project references, not necessary when consumed as a NuGet package.